src/specimens/Html.js

```diff
@@ -2,12 +2,33 @@ import React, { PropTypes } from 'react';
 import {catalogShape} from '../CatalogPropTypes';
 import Radium from 'radium';
 import Frame from '../components/Frame/Frame';
+import Hint from '../specimens/Hint';
 import Specimen from '../components/Specimen/Specimen';
 import HighlightedCode from '../components/HighlightedCode/HighlightedCode';
 import runscript from '../utils/runscript';
 
 const PADDING = 3;
 const SIZE = 20;
+const SCREENSIZES = [
+  {name: 'small', width: 360, height: 640},
+  {name: 'medium', width: 1024, height: 768},
+  {name: 'large', width: 1440, height: 900},
+  {name: 'xlarge', width: 1920, height: 1080}
+];
+const INSTRUCTION =
+`theme: {
+  //...
+  devices: [
+    {name: 'watch-42mm', width: 312, height: 390}
+  ],
+  //...
+},
+`;
+
+const getScreensizes = (customSizes) => [
```

Comment:
Again, handle the merging in the configuration, since we don't have different sizes per specimen